cpp/src/slice2swift/SwiftUtil.cpp

```diff
@@ -1523,19 +1523,14 @@ SwiftGenerator::writeMemberwiseInitializer(IceUtilInternal::Output& out,
                                            const DataMemberList& baseMembers,
                                            const DataMemberList& allMembers,
                                            const ContainedPtr& p,
-                                           bool local,
+                                           bool /*local*/,
```

Comment:
Would be better to remove this parameter, and fix the places where this is called.

Reply:
I didn't want to be too invasive :-P

I will have a look at it.